landoapi/transplants.py

```diff
@@ -301,6 +303,47 @@ def warning_wip_commit_message(*, revision, **kwargs):
         return "This revision is marked as a WIP. Please remove `WIP:` before landing."
 
 
+@RevisionWarningCheck(8, "Repository is under a soft code freeze.", True)
+def warning_code_freeze(*, repo, **kwargs):
+    repo_fields = repo.get("fields", {})
+    short_name = repo_fields.get("shortName")
+    if short_name == "mozilla-central":
+        message = ""
```

Comment:
`message` should not be blank -- if there is an unhandled case, the message should be explicit about that. Cleaning up / refactoring the try/except block should make it clearer as to what is happening with `message`.